src/overview/components/Overview.jsx

```diff
@@ -18,7 +19,7 @@ class Overview extends React.Component {
                 ref='inputQuery'
             >
             </input>
-            <ResultList searchResult={this.props.searchResult} />
+            {this.props.loading==0 ? <ResultList searchResult={this.props.searchResult} />: <Loader />  }
```

Comment:
I am expecting you mean something like
```
{this.props.loading==0 ? <ResultList searchResult={this.props.searchResult} />: <Loader />  }
```
But I don't think this will work(it doesn't). The control loop takes a value which is boolean and cannot be int. Unless we make a complex workaround this.